shared/react-native/android/app/src/main/java/io/keybase/ossifrage/modules/ShareFiles.java

```diff
@@ -31,11 +36,28 @@ public String getName() {
     @ReactMethod
     public void share(String uriPath, String mimeType, Promise promise) {
         File file = new File(uriPath);
-        Uri fileUri = FileProvider.getUriForFile(reactContext, "io.keybase.ossifrage.fileprovider", file);
-
-        Intent intent = new Intent(Intent.ACTION_SEND)
-                .putExtra(Intent.EXTRA_STREAM, fileUri)
-                .setType(mimeType);
+        Intent intent = new Intent(Intent.ACTION_SEND).setType(mimeType);
+        if (mimeType.contains("text/")) {
```

Comment:
Should this be `mimeType. startsWith(...)`?

Reply:
Sure